index.d.ts

```diff
@@ -521,6 +521,17 @@ export declare interface TracerOptions {
      */
     blockedTemplateJson?: string,
   };
+
+  /**
+   * Configuration of ASM Remote Configuration
+   */
+  remoteConfig?: {
+    /**
+     * Specifies the remote configuration polling interval in seconds
+     * @default 5
+     */
+    pollInterval?: number,
```

Comment:
You also need to add an entry to `docs/test.js` when adding new programmatic conf

Reply:
basically just add that line 87
```js
remoteConfig: {
  pollInterval: 5
}
```

Reply:
Done!